src/deploy-tes-on-azure/Deployer.cs

```diff
@@ -1306,6 +1245,63 @@ await postgreSqlDnsZone
             return server;
         }
 
+        private string GetFormattedPostgresqlUser()
+        {
+            if (!configuration.ProvisionPostgreSqlOnAzure.GetValueOrDefault())
+            {
+                return string.Empty;
+            }
+
+            if (configuration.UsePostgreSqlSingleServer)
+            {
+                return $"{configuration.PostgreSqlTesUserLogin}@{configuration.PostgreSqlServerName}";
+            }
+            else
+            {
+                return configuration.PostgreSqlTesUserLogin;
+            }
+        }
+
+        private string GetCreateTesUserString()
+        {
+            return $"CREATE USER {configuration.PostgreSqlTesUserLogin} WITH PASSWORD '{configuration.PostgreSqlTesUserPassword}'; GRANT ALL PRIVILEGES ON DATABASE {configuration.PostgreSqlTesDatabaseName} TO {configuration.PostgreSqlTesUserLogin};";
+        }
+
+        private string GetPostgreSQLCreateUserCommand(bool useSingleServer, string dbName, string sqlCommand)
+        {
+            if (useSingleServer)
+            {
+                return $"PGPASSWORD={configuration.PostgreSqlAdministratorPassword} psql -U {configuration.PostgreSqlAdministratorLogin}@{configuration.PostgreSqlServerName} -h {configuration.PostgreSqlServerName}.postgres.database.azure.com -d {dbName}  -v sslmode=true -c \"{sqlCommand}\"";
+            }
+            else
+            {
+                return $"psql postgresql://{configuration.PostgreSqlAdministratorLogin}:{configuration.PostgreSqlAdministratorPassword}@{configuration.PostgreSqlServerName}.postgres.database.azure.com/{dbName} -c \"{sqlCommand}\"";
+            }
+        }
+
+        private Task ExecuteQueriesOnAzurePostgreSQLDbFromK8()
+            => Execute(
+                $"Executing scripts on postgresql...",
+                async () =>
+                {
+                    var tesScript = GetCreateTesUserString();
+                    var serverPath = $"{configuration.PostgreSqlServerName}.postgres.database.azure.com";
+                    var adminUser = configuration.PostgreSqlAdministratorLogin;
+
+                    if (configuration.UsePostgreSqlSingleServer)
+                    {
+                        adminUser = $"{configuration.PostgreSqlAdministratorLogin}@{configuration.PostgreSqlServerName}";
+                    }
+
+                    var commands = new List<string[]> {
+                        new string[] { "bash", "-lic", $"echo {configuration.PostgreSqlServerName}.postgres.database.azure.com:5432:{configuration.PostgreSqlTesDatabaseName}:{adminUser}:{configuration.PostgreSqlAdministratorPassword} >> ~/.pgpass" },
+                        new string[] { "chmod", "0600", "/home/tes/.pgpass" },
```

Comment:
Does `~/.pgpass` not work here? While home directories are nearly always in a known subdirectory of the `/home` directory, that's not always the case, and if something really weird happens this won't work as expected.

Reply:
@jsaun any thoughts?

Reply:
Resolved by @jsaun